templates/common/_base/units/ovs-notification.yaml

```diff
@@ -0,0 +1,21 @@
+name: ovs-notification.service
+enabled: {{if eq .NetworkType "OVNKubernetes"}}true{{else if eq .NetworkType "OpenShiftSDN"}}true{{else}}false{{end}}
+contents: |
+  [Unit]
+  Description=Creates a file to let the OVS pod know OVS is running on systemd.
+  # Don't write the notification before OVS starts
+  Requires=openvswitch.service
+  Wants=NetworkManager-wait-online.service
+  After=NetworkManager-wait-online.service openvswitch.service
+  Before=network-online.target kubelet.service crio.service node-valid-hostname.service
```

Comment:
Why should we care about most of those services? If we run before kubelet then we run before the sdn-ovs pod and we're good, right?

Reply:
Did that to be consistent with the ovs-configuration.service.yaml, but you're right.

I'm pretty sure we don't need the node-valid-hostname.service and crio. Regarding the network-online.target, I think it's necessary so that the network-online.target isn't complete until this is finished.